ir/src/constraint_builder/integrity_constraints/mod.rs

```diff
@@ -4,9 +4,10 @@ use super::{
     ListFoldingValueType, NamedTraceAccess, SemanticError, Symbol, SymbolType, Variable,
     VariableType, VectorAccess, CURRENT_ROW,
 };
+// TODO: Move this to air-script-core
+use parser::ast::ConstraintType;
```

Comment:
Does it make sense to just put this in `air-script-core` now?

Reply:
Sorry, I didn't realize then that we will have to move all it's dependencies to `air-script-core` (which would essentially be almost everything in parser). So, I have just imported from `super::ast` instead. Please let me know if you disagree with the change.

Reply:
Makes sense!